crawl-ref/source/aptitudes.h

```diff
@@ -1332,6 +1332,7 @@ static const species_skill_aptitude species_skill_aptitudes[] =
     APT(SP_OCTOPODE,        SK_EVOCATIONS,      1),
 
 #if TAG_MAJOR_VERSION == 34
+    // Need to keep this as long as the species are still around
```

Comment:
What does "still around" mean, here? As I understand it you have removed the species; do you mean until there are no save files anywhere with Djinni? If so, I think be more explicit in the comment.